pyutilib/misc/redirect_io.py

```diff
@@ -49,6 +51,18 @@ def reset_redirect():
         sys.stderr = _old_stderr.pop()
 
 
+@contextmanager
+def redirected_stdout(output=None):
```

Comment:
What do you think of renaming this `capture_output`?  The rationale is that `setup_redirect` captures both stdout and stderr, plus it would read a little better:

```
with capture_output() as output:
    # do something
text = output.getvalue()
```